packages/akar/src/a-popper/a-popper-arrow.vue

```diff
@@ -0,0 +1,80 @@
+<script lang="ts">
+import type { APrimitiveProps } from '~~/a-primitive';
+import type { ArrowProps } from '~~/shared/component/a-arrow.vue';
+
+import type { Side } from './utils';
+
+const OPPOSITE_SIDE: Record<Side, Side> = {
+  bottom: 'top',
+  left: 'right',
+  right: 'left',
+  top: 'bottom',
+};
+
+export interface APopperArrowProps extends ArrowProps, APrimitiveProps {}
+</script>
+
+<script setup lang="ts">
+import { computed } from 'vue';
+
+import { useForwardExpose } from '~~/shared';
+import AArrow from '~~/shared/component/a-arrow.vue';
+
+import { injectPopperContentContext } from './a-popper-content.vue';
+
+defineOptions({
+  inheritAttrs: false,
+});
+
+withDefaults(
+  defineProps<APopperArrowProps>(),
+  { as: 'svg' },
+);
+
+const { forwardRef } = useForwardExpose();
+const contentContext = injectPopperContentContext();
+
+const baseSide = computed(() => OPPOSITE_SIDE[contentContext.placedSide.value]);
+</script>
+
+<template>
+  <span
+    :ref="(el: HTMLElement) => {
+      contentContext.onArrowChange(el)
+      return undefined
+    }"
+    :style="{
```

Comment:
Can we move this to the script section?

Reply:
I've modified the code. 

Kindly inform me if it's not enough, otherwise, please resolve the conversation.